### PR TITLE
[frio] Word-wrap for long lines

### LINKED ISSUE #3837: Firefox v64+: Long lines of code content don't wrap
If you use the markup [code][/code] it produces a very ugly output because it doesn't do wordwrap. I understand that it would break the concept, but this looks like the page has an error. So maybe add scrollbars to such an element.
00temp

### PR DESCRIPTION
**Pull request by nupplaphil**
Should fix #3837 

bildschirmfoto von 2019-03-02 16-03-30

### CODE REVIEW

File: view/theme/frio/css/style.css [hunk 1]
```diff
@@ -1620,6 +1620,7 @@ textarea#profile-jot-text:focus + #preview_profile-jot-text {
 .panel .panel-body {
     padding: 15px;
     font-size: 14px;
+	word-wrap: break-word;
```
Comment (MrPetovan) [inline view/theme/frio/css/style.css]: Wrong indenting compared to the rest of the file (I'm sorry).

Comment (nupplaphil) [reply view/theme/frio/css/style.css]: So 100% wrong code haha

**Comment by MrPetovan** [comment]
Have you tested on multiple browsers? At least Chrome and Firefox?

**Comment by nupplaphil** [comment]
- Firefox 65 (x64)
- Chromium 72
- Opera 58

**Comment by MrPetovan** [comment]
I'm asking because someone submitted a similar patch not long ago, but it has to be reverted.

**Pull request merged**